lib/web/apiserver.go

```diff
@@ -1546,6 +1546,14 @@ func (h *Handler) siteSessionGet(w http.ResponseWriter, r *http.Request, p httpr
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+
+	// DELETE IN: 4.5.0
```

Comment:
This should probably be 5.0.0 - we aren't currently planning a 4.5.0.

Reply:
revised